stdlib/ballerina-database/src/main/ballerina/mysql/Package.md

```diff
@@ -0,0 +1,29 @@
+## Package overview
+
+This package provides the functionality required to access and manipulate data stored in MySQL database. 
+
+### Endpoint 
+
+To access a database, you must first create an `endpoint`, which is a virtual representation of the physical endpoint of the MySQL database that you are trying to connect to. Create an endpoint of the MySQL client type (i.e., `mysql:Client`) and provide the necessary connection parameters. This will create a pool of connections to the given MySQL database. A sample for creating an endpoint with MySQL client can be found below. 
+
+### Database operations
+
+Once the endpoint is created, database operations can be executed through that endpoint. This package provides support for creating tables and executing stored procedures. It also supports selecting, inserting, deleting, updating, and batch updating data. For more details on the supported actions refer the `jdbc` package. Details of the SQL data types and query parameters relevant for these database operations could be found in the documentation of `sql` package. 
```

Comment:
Change "refer the `jdbc` package" to "refer to the `jdbc` package"
Change "relevant for" to "relevant to"
Change "could be found" to "can be found" 
Change "of `sql` package" to "of the `sql` package"